### ISSUE TITLE
Ownership transfer when Owner is banned

### ISSUE THREAD

**Issue by josfire18** [EarthMC/EarthMC.net#58]
**Exact wording of the rule**
If a nation leader is banned, members of said nation can contact mods to have the next in line promoted to leadership. The members will have to agree on the next in line before contacting any mods.

**How will this rule affect the community**
When a nation's leader is banned, the nation will fall since no one can take ownership after the fact. This is such a bad rule because the leader could do something and it would affect everyone, despite no one else being involved but the leader.

**Do you think this will benefit both the staff and the players**
Yes. Fewer ticket requests and complaining to staff and players don't have to lose their entire nation when one person gets banned. Why punish everyone for the mistakes of one?

**Please use <https://github.com/EarthMC/Issue-Tracker> for issues other than the EarthMC.net website**

**Comment by daawesomemc** [comment]
Yeah, same thing with towns. Imagine you log on, find your entire town destroyed and all your stuff stolen/looted, and then you find out that the mayor/nation leader got banned, the town disbanded, and everything raided.

**Comment by josfire18** [comment]
Yeah and threats of mutes for speaking up to this bs is even more reason to make a change.

Thank you,
Joey Rockwood
IT
Ext. 1134
________________________________

**Comment by DasCodes** [comment]
I don't think this is necessary as when a player gets inactive (which they will be most likely after a permanent ban unless appealed and unbanned) the leadership of the town is passed down to another player in the town. When the new player becomes the mayor of the town, if it's the capital of that nation, he'll get the leadership of the nation as well. Though I'm not sure how towny transfers the ownership if it's the seniority of the player or something else.

**Comment by daawesomemc** [comment]
Never knew that, ty!

**Comment by DasCodes** [comment]
Also to note here, if we to implement a function, if a nation leader or mayor gets banned, the leadership will pass down to the next keen, there will be more issues. Suppose the player was banned for some reason and appealed, and got unbanned. He'll most likely want the leadership back. What happens when the new leader doesn't wanna give up the leadership which will end up in a conflict and more work for us. So I find it best to leave the leadership changing system to towny.

**Comment by josfire18** [comment]
He's been permabanned. There is no leader right now. I submitted a ticket and they said they don't do that currently and I should submit this request.

Thank you,
Joey Rockwood
IT
Ext. 1134
________________________________

**Comment by josfire18** [comment]
This would be for permabans like our current situation where the mods have told him he is not going to be unbanned.

Thank you,
Joey Rockwood
IT
Ext. 1134
________________________________

**Comment by DasCodes** [comment]
Isn't there anyone who plays actively in that town?

**Comment by josfire18** [comment]
There are many.

Thank you,
Joey Rockwood
IT
Ext. 1134
________________________________

**Comment by josfire18** [comment]
But I don't know if they left or not.

Thank you,
Joey Rockwood
IT
Ext. 1134
________________________________

**Comment by DasCodes** [comment]
In that case, the nation won't be disbanded. As long as there is a single active player in a town, the town won't fall. But yes, we don't transfer the ownership of a nation or a town currently.

**Comment by josfire18** [comment]
I just checked and death seems to be the last active member. The nation is very active though.

Thank you,
Joey Rockwood
IT
Ext. 1134
________________________________

**Comment by shirazmatas** [comment]
**Remember to express your opinions on the suggestion by using the emojis next to the original post**


> 
> 
> I don't think this is necessary as when a player gets inactive (which they will be most likely after a permanent ban unless appealed and unbanned) the leadership of the town is passed down to another player in the town. When the new player becomes the mayor of the town, if it's the capital of that nation, he'll get the leadership of the nation as well. Though I'm not sure how towny transfers the ownership if it's the seniority of the player or something else.

I think this is unfair because it takes 42 days before it's moved. Would be better if it moved to a councillor manually.
